pkg/identityserver/identityserver_test.go

```diff
@@ -312,3 +312,15 @@ func getIdentityServer(t *testing.T) (*IdentityServer, *grpc.ClientConn) {
 func testWithIdentityServer(t *testing.T, f func(is *IdentityServer, cc *grpc.ClientConn)) {
 	f(getIdentityServer(t))
 }
+
+func reverse(s string) string {
```

Comment:
So the only use for this is to generate a different string from a constant in 5 test cases?
IMO, that's completely redundant - just define a second constant in test case and use that explicitly. That's easier to maintain, clearer and simpler to debug.

Reply:
No, those are generated by the populators. The reason for this change is that the populators sometimes generated IDs that were already 36 long, and then adding "mod" would make the IDs invalid.